RecoTracker/IterativeTracking/python/PixelPairStep_cff.py

```diff
@@ -165,6 +165,23 @@
      ptMin = 0.05,
      originRadius = 0.2,
 ))
+#include commented lines from above in pp_on_XY eras; global seeds (A) are not used in this era b/c timing
+from Configuration.Eras.Modifier_pp_on_XeXe_2017_cff import pp_on_XeXe_2017
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+for e in [pp_on_XeXe_2017, pp_on_AA_2018]:
+      e.toModify(pixelPairStepTrackingRegionsSeedLayersB, layerList = [
+      "BPix1+BPix2", "BPix1+BPix3", "BPix1+BPix4", "BPix2+BPix3", "BPix2+BPix4","BPix3+BPix4",
```

Comment:
this adds quite a few more layer pairs.
Also, IIUC, the pixelPairStepTrackingRegionsSeedLayersB becomes active only for tracker failure areas.
Was this code timing tested on some tracker failure scenarios?

@makortel please take a look to confirm that the strategy in this PR is appropriate.
Thank you.

Reply:
@slava77 It looks to me that the changes are according to the comment above
```python
#include commented lines from above in pp_on_XY eras; global seeds (A) are not used in this era b/c timing
```
i.e. the layer pairs of the global A seeding are moved here, and A seeding is disabled. I'd assume it to be faster than the default A+B seeding. Of course then there is nothing to back up the pixel triplets in the barrel-forward transition region.